config/locales/idv/es.yml

```
@@ -114,9 +114,9 @@ es:
         timeout: Nuestra solicitud para verificar tu información ha caducado. Vuelve a
           intentarlo.
         warning:
-          attempts:
-            one: Por razones de seguridad, dispone de un solo intento.
-            other: Por razones de seguridad, le quedan %{count} intentos.
+          attempts_html:
+            one: Por razones de seguridad, dispone de <strong>un solo intento</strong>.
+            other: Por razones de seguridad, le quedan <strong>%{count} intentos</strong>.
```

I see that the discrepancy was already there in the text you edited, but I think these two sentences should have the same form, the way they do on lines 296 and 297. We should probably choose between razones (reasons) and motivos (intentions) and make that consistent too.
```suggestion
            one: Por razones de seguridad, le quedan <strong>un solo intento</strong>.
            other: Por razones de seguridad, le quedan <strong>%{count} intentos</strong>.
```